language/move-prover/bytecode-to-boogie/test_mvir/test-aborts-if.mvir

```diff
@@ -83,8 +83,119 @@ module TestAbortIf {
                       return;
```

Comment:
@wrwg: I initially tried to put these new tests in test-arithmetic.mvir, but then I noticed the test harness runs that file in no-verify mode. Is this file the best place for these arithmetic op tests?

Reply:
This is a bit messy right now. We have two sets of tests: those which are intended to pass verification (modulo expected errors) and those which aren't. The later is a bit of legacy (perhaps we want all tests to pass verification?), and they are in translator_tests.rs.

This regressed because folks where putting verification tests in translator_tests, and I did not pay attention during review.

Bottom line: please put all tests which should have verification into prover_tests.rs. Those new tests which include verification should be as focused as possible. That is instead of one mvir file for all arithmetics, one mvir file for each operator being tested. The reason for this is not only cleaner test organization and easier failure analysis, but also to isolate prover state, and -- if the test produces verification errors -- to easier isolate those (via the `//!` syntax) in absence of good location reporting.

Reply:
Thanks for taking the time to explain this plan! I'll create a couple of new files.

In addition, I think it would be great if writing a new test only required adding a .mvir file (i.e., no changes to prover_tests.rs). One way we could achieve this is by writing a harness that first verifies the Move stdlib, then looks at `test_mvir` and verifies each file there in an env that contains the stdlib.